frontend/app/routes/projects/create-new.js

```diff
@@ -0,0 +1,23 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { hash } from 'rsvp';
+
+class Model {
+  @tracked saved = false;
+
+  save() {
+    this.saved = true;
+  }
+}
+
+export default class ProjectsCreateNewRoute extends Route {
+  @service store;
+  model() {
+    return hash({
+      model: new Model(),
+      dependencies: this.store.findAll('dependency'),
+      dependencyVersions: this.store.findAll('dependencyVersion'),
+    });
+  }
```

Comment:
This was moved to the controller, delete it

Reply:
resolved as of https://github.com/adfinis/Outdated/pull/14/commits/a3cd8323af2efeea6358f7091eac4b2b2180148e